pkgs/development/compilers/gcl/default.nix

```diff
@@ -26,11 +26,13 @@ assert stdenv.cc.libc != null;
 
 stdenv.mkDerivation rec {
   pname = "gcl";
-  version = "2.6.14";
+  version = "2.7.0pre34";
 
-  src = fetchurl {
-    url = "mirror://gnu/gcl/gcl-${version}.tar.gz";
-    hash = "sha256-CfNBfFEqoXM6Y4gJ06Y6wpDuuUSL6CeV9bZoG9MHNFo=";
+  src = fetchgit {
+    # url = "mirror://gnu/gcl/gcl-${version}.tar.gz";
+    url = "git://git.savannah.gnu.org/gcl.git/";
```

Comment:
I think there's a fetcher for this which uses the http archive endpoint https://github.com/hraban/nixpkgs/blob/8ac4a94f1b25a84c8bb5498cb4204dc5c2dd27ab/pkgs/top-level/all-packages.nix#L638 it seems unused in nixpkgs but if it works this would be a good occasion to use it.